app/views/analytics/history.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= t('analytics.history.headline') %></h1>
+
+<% if @mode == "borrowed_by_me" %>
+    <div class="mx-auto btn btn-dark">
+        <%= link_to t('items.headlines.my_items'), analytics_history_borrowed_from_me_path, class: "text-decoration-none text-white"%>
+    </div>
```

Comment:
In general I understand what the respective lists both show, but I think for clarity it might be nice to have a little descriptor summarizing what the user is looking at at a glance, seeing as. Yes, the button highlight helps, but seeing as otherwise nothing on the page gives away which "mode" you are currently viewing this may be a sensible, inexpensive addition.

Of course this would ideally need to be done for the mode below as well.

```suggestion
    </div>
    <hr>
    <p><i>Items that I have borrowed in the past</i></p>
```